Divine/CLI/CommandLineActions.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Divine.Enums;
+using LSLib.LS;
+
+namespace Divine.CLI
+{
+    internal class CommandLineActions
+    {
+        public static string SourcePath;
+        public static string DestinationPath;
+        public static string ConformPath;
+
+        public static Game Game;
+        public static LogLevel LogLevel;
+        public static ResourceFormat InputFormat;
+        public static ResourceFormat OutputFormat;
+        public static PackageVersion PackageVersion;
+        public static Dictionary<string, bool> GraphicsOptions;

```

Comment:
GR2Options or ModelOptions (as it has options for both GR2 and DAE, and possibly more later) would be a better fit, as these are not exactly "graphics" options

Reply:
Okay.

Reply:
By the way, my reasoning for "Graphics" was that the acronym GR stands for Graphics Resource, but I've made the change you requested. Not committed yet.